lib/cpp/test/TransportTest.cpp

```diff
@@ -1016,7 +1019,7 @@ struct global_fixture {
   }
 };
 
-BOOST_GLOBAL_FIXTURE(global_fixture)
+BOOST_GLOBAL_FIXTURE(global_fixture);
```

Comment:
is this change necessary for boost 1.59? in previous versions it causes warning "extra ;"